### DISCUSSION TITLE
Linux启动求助

### DISCUSSION THREAD

**Discussion by TsukasaYahane** [nonebot/discussions#458] [问答] [answered]
编写完新插件后，在本地console运行可正常使用。丢到wsl的Ubuntu安装依赖环境启动后，提示报错bot.py疑似出现循环引用
![image](https://github.com/nonebot/discussions/assets/121178437/590a9342-75a9-4a43-bb58-f22cff437243)
![image](https://github.com/nonebot/discussions/assets/121178437/1d70d265-53ae-456e-9abe-4485556d67c0)

**Comment by yanyongyu** [comment, marked as answer]
你的环境配错了吧，同时安装了nb1和nb2，请将两者都卸载掉后安装nb2

```bash
pip uninstall nonebot nonebot2
pip install nonebot2
```

**Reply by TsukasaYahane** [reply]
已解决 谢谢大佬解答
